docs/en/04_Changelogs/3.2.0.md

```diff
@@ -17,6 +17,7 @@
  * `Mailer` no longer calls `xml2raw` on all email subject line, and now must be passed in via plain text.
  * `ErrorControlChain` now supports reload on exceptions
  * `FormField::validate` now requires an instance of `Validator`
+ * Admin URL can now be configured via custom Director routing rule
```

Comment:
It might need to be 4.0 if it breaks modules which rely on the admin url being fixed as 'admin'.